net.sourceforge.efaxgtk.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: net.sourceforge.efaxgtk
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: efax-gtk
+rename-desktop-file: efax-gtk.desktop
+rename-icon: efax-gtk
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=session-bus
```

Comment:
more granular dbus permissions are needed.

Reply:
You need at least `--talk-name=org.gtk.vfs.*` for GVFS, and also `--filesystem=xdg-run/gvfs`